src/ert/_c_wrappers/enkf/enkf_main.py

```diff
@@ -274,75 +269,28 @@ def loadFromForwardModel(
         fs.sync()
         return nr_loaded
 
-    def create_ensemble_experiment_run_context(
-        self, iteration: int, active_mask: List[bool] = None, source_filesystem=None
-    ) -> RunContext:
-        """Creates an ensemble experiment run context
-        :param fs: The source filesystem, defaults to
-            getEnkfFsManager().getCurrentFileSystem().
-        :param active_mask: Whether a realization is active or not,
-            defaults to all active.
-        """
-        return self._create_run_context(
-            iteration=iteration,
-            active_mask=active_mask,
-            source_filesystem=source_filesystem,
-            target_fs=None,
-        )
-
-    def create_ensemble_smoother_run_context(
-        self,
-        iteration: int,
-        target_filesystem,
-        active_mask: List[bool] = None,
-        source_filesystem=None,
-    ) -> RunContext:
-        """Creates an ensemble smoother run context
-        :param fs: The source filesystem, defaults to
-            getEnkfFsManager().getCurrentFileSystem().
-        """
-        return self._create_run_context(
+    def create_ensemble(self, case_name, active_realizations, iteration) -> RunContext:
+        return RunContext(
+            sim_fs=self.storage_manager.add_case(case_name),
+            path_format=self.resConfig().preferred_job_fmt(),
+            format_string=self.getModelConfig().getRunpathFormat().format_string,
+            runpath_file=self.resConfig().runpath_file,
+            initial_mask=active_realizations,
+            global_substitutions=dict(self.get_context()),
             iteration=iteration,
-            active_mask=active_mask,
-            source_filesystem=source_filesystem,
-            target_fs=target_filesystem,
         )
 
-    def _create_run_context(
-        self,
-        iteration: int = 0,
-        active_mask: List[bool] = None,
-        source_filesystem=None,
-        target_fs=None,
-    ) -> RunContext:
-        if active_mask is None:
-            active_mask = [True] * self.getEnsembleSize()
-        if source_filesystem is None:
-            source_filesystem = self.getCurrentFileSystem()
-        realizations = list(range(len(active_mask)))
-        paths = self.runpaths.get_paths(realizations, iteration)
-        jobnames = self.runpaths.get_jobnames(realizations, iteration)
-        for realization, path in enumerate(paths):
-            self.substituter.add_substitution("<RUNPATH>", path, realization, iteration)
-        for realization, jobname in enumerate(jobnames):
-            self.substituter.add_substitution(
-                "<ECL_BASE>", jobname, realization, iteration
-            )
-            self.substituter.add_substitution(
-                "<ECLBASE>", jobname, realization, iteration
-            )
+    def load_ensemble(self, case_name, active_realizations, iteration) -> RunContext:
```

Comment:
I understand we need to discuss the `Ensemble` vs `RunContext` name.. I don't think it's sensible to rename functions to saying they return `ensemble` if they return an instance of `RunContext`.. It will only make it difficult for future developers.

If we are to rename `RunContext` to `Ensemble` we should figure out what the current `Class Ensemble` is and rename that as well. Also, we should make sure that the new `Ensemble` does not know more than it needs, e.g. should it know about `iteration`? Are there other things that makes sense the `Ensemble` class should know?